onboarding/managers.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Onboarding Responsibilities for Managers of New Hires
+description: Things for managers to remember during the early days of onboarding
+---
+
+## Onboarding For Managers
+
+### Before Day 1
+
+- [ ] Choose a peer mentor for the new hire
+- [ ] Choose a spirit guide for the new hire (consult with People Ops)
+- [ ] Set first-week schedule
+- [ ] Invite new hire to all engineering-wide meetings
```

Comment:
We should also have the manager contact the new hire's product team lead and have them invited to team-specific events (retrospectives, sprint planning, knowledge shares, etc).